submission/persistence/src/test/kotlin/ac/uk/ebi/biostd/persistence/converters/EpochDateConverterTest.kt

```diff
@@ -0,0 +1,26 @@
+package ac.uk.ebi.biostd.persistence.converters
+
+import ebi.ac.uk.functions.secondsToInstant
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.time.ZoneOffset
+
+class EpochDateConverterTest {
```

Comment:
I think we still need to unit test suites.

Reply:
Done